src/modules/drawing.js

```diff
@@ -0,0 +1,123 @@
+import assign from 'object-assign'
+
+const debug = require('debug')('wololobot:drawing')
+
+const removeFromArr = (arr, el) => {
+  let i
+  while ((i = arr.indexOf(el)) !== -1) {
+    arr.splice(i, 1)
+  }
+  return arr
+}
+
+export default function(opts) {
+
+  opts = assign({
+    subChances: 2     // Number of tickets for subs
+  , normalChances: 1  // Number of tickets for non-subs
+  }, opts)
+
+  let entrants = []
+  let winners = []
+  let nEntrants = 0
+  let open = false
+
+  function pickRandom() {
+    debug(`Picking a random entrant out of ${entrants}`)
+    let rand = entrants[Math.floor(Math.random() * entrants.length)]
+    winners.push(rand)
+    entrants = removeFromArr(entrants, rand)
+  }
+
+  function winnersStr() {
+    if (winners.length === 0) {
+      return 'There are no winners!'
+    } else if (winners.length === 1) {
+      return `The winner is ${winners[0]}`
+    }
+    let out = 'The winners are: '
+    winners.forEach((winner) => {
+      out += `${winner}, `
+    })
+    return out.slice(0, -2)
+  }
+
+  return function drawing(bot) {
+    bot.command('!play', (message) => {
+      if (!open) {
+        return
+      }
+      message.user =  message.user.toLowerCase()
+      if (entrants.indexOf(message.user) !== -1) {
+        return
+      }
+      nEntrants++
+      if (message.tags.subscriber === '1') {
+        for (let i = 0; i < opts.subChances; i++) {
+          entrants.push(message.user)
+        }
+      } else {
+        for (let i = 0; i < opts.normalChances; i++) {
+          entrants.push(message.user)
+        }
+      }
+    })
+
+    bot.command('!draw open'
+                , { rank: 'mod' }
+                , (message) => {
+      entrants = []
+      open = true
+      nEntrants = 0
+      bot.send('New drawing opened! To enter, type !play. Subscribers have double the chances to' +
```

Comment:
minor: it's not really always double with configurable chances. maybe `${n} times`  or so (or you can just not care)